source/pastemyst/data/config.d

```diff
@@ -95,60 +93,64 @@ static this()
     {
         import std.file : exists;
         import std.exception : enforce;
+        import std.format : format;
 
-        if (!exists("config.yaml"))
-        {
-            throw new Exception("missing config.yaml");
-        }
+        // default configuration file
+        string configName = "config.yaml";
+        enforce(
+            // if doesn't exists, fallback to .yml
+            exists(configName) || exists(configName = "config.yml"),
+            format!"missing %s"(configName));
```

Comment:
could you rewrite this to make more sense and look simpler since theres a lot going on in just this one enforce call. generally you shouldnt change variables inside function calls parameters since it looks confusing.

Reply:
I thought this was simpler with an explanatory comment, but I'll change it.